src/components/molecular/UserStatus.vue

```diff
@@ -46,6 +42,18 @@ export default {
         })
         .then(() => {
           localStorage.removeItem('sessionToken')
+          localStorage.removeItem('workspace')
```

Comment:
(partially) same code as in `LoginForm`. That proves the worth to extract it into a separate script file :smile:

Reply:
done bffe2d1